qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -147,22 +147,47 @@ def _update_status_queue_info(self) -> None:
                 self._overall_status = temp_status
                 return
 
-    def compiled_circuits(self) -> List[qiskit.QuantumCircuit]:
-        """Get the compiled circuits that were submitted for this job.
+    def _get_circuits(self, circuit_type: str) -> List[qiskit.QuantumCircuit]:
+        """Retrieves the corresponding circuits to `circuit_type`.
+
+        Args:
+            circuit_type: The kind of circuits to retrieve. Either "input_circuits" or
+                "compiled_circuits".
```

Comment:
```suggestion
            circuit_type: The kind of circuits to retrieve. Either "input_circuit" or
                "compiled_circuit".
```

Reply:
(based on how the function is called later on in this file)

Reply:
Good catch! Fixed now